.github/workflows/main.yml

```diff
@@ -0,0 +1,27 @@
+---
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  test-integration:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: opensafely-core/research-action@v2
+
+  tag-new-version:
+    needs: [test-integration]
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Tag new version
+        uses: mathieudutour/github-tag-action@d745f2e74aaf1ee82e747b181f7a0967978abee0
```

Comment:
On this line as this commit is the `v6.0` tag I wondered whether it's preferable to refer to that with `@v6.0` 

https://github.com/mathieudutour/github-tag-action/tree/releases/v6.0

Reply:
Yes and no 🙂 The tag is certainly easier to interpret than the SHA. However, tags are mutable, but SHAs are not. mathieudutour could point v6.0 to another commit, and then we'd be running a different version of github-tag-action.